src/components/patients/patientTable/index.tsx

```diff
@@ -17,20 +17,23 @@ const PatientTable = () => {
   const [patients, setPatients] = React.useState([] as Patient[]);
 
   const renderPatientAttribute = (
-    attribute: "firstName" | "lastName" | "age",
+    attribute: "id" | "firstName" | "lastName" | "age",
     index: number
   ) => (
     <Cell>
       <React.Fragment>
         <Link to={`${ROUTE_PATIENT}/${patients[index].id}`}>
-          {patients[index][attribute]}
+          {patients[index][attribute] || "unknown"}
```

Comment:
when is `patients[index][attribute]` null ?

Reply:
First name, last name and birth date is not mandatory in FHIR
https://www.hl7.org/fhir/patient.html